src/scss/components/_layout-header-card.scss

```diff
@@ -0,0 +1,107 @@
+.ucf-news.header-card-layout {
```

Comment:
Can we add these styles to src/scss/components/_news.scss instead of a new file?